servicetalk-http-api/src/main/java/io/servicetalk/http/api/StreamingHttpRequest.java

```diff
@@ -140,10 +168,12 @@
     /**
      * Returns a {@link StreamingHttpRequest} with its underlying payload transformed to {@link Object}s,
      * with access to the trailers.
+     * @deprecated use {@link #transform(TrailersTransformer)}.
      * @param trailersTransformer {@link TrailersTransformer} to use for this transform.
      * @param <T> The type of state used during the transformation.
      * @return {@code this}
      */
+    @Deprecated
     <T> StreamingHttpRequest transformRaw(TrailersTransformer<T, Object> trailersTransformer);
```

Comment:
This method is still used in `ContentHeadersTest`, please update the test.

Reply:
My thought process here -> while the deprecated method still exists if there are tests dedicated to the deprecated methods we should leave them, so the behavior is still verified. I plan to remove all usage when the deprecated methods are removed.

Reply:
Makes sense, thanks for clarifying!